front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -163,56 +176,46 @@ const SimulationReportSheet = ({
-                        <View style={styles.convoyAndRoute.stopTableOpWidth}>
-                          <TD style={styles.convoyAndRoute.stopTableOpColumn}>
-                            {step.name || 'Unknown'}
-                          </TD>
-                        </View>
-                        <View style={styles.convoyAndRoute.stopTableChWidth}>
-                          <TD style={styles.convoyAndRoute.stopTableChColumn}>{step.ch}</TD>
-                        </View>
-                        <View style={styles.convoyAndRoute.stopTableEndWidth}>
-                          <TD style={styles.convoyAndRoute.stopTableItalicColumn}>
-                            {isLastStep ? t('asap') : ''}
-                          </TD>
-                        </View>
-                        <View style={styles.convoyAndRoute.stopTableStartWidth}>
-                          <TD style={styles.convoyAndRoute.stopTableStartColumn}>
-                            {isFirstStep ? step.stopEndTime : ''}
-                          </TD>
-                        </View>
-                        <View style={styles.convoyAndRoute.stopTableStopTypeWidth}>
-                          <TD style={styles.convoyAndRoute.stopTableItalicColumn}>
-                            {isFirstStep || isLastStep
-                              ? t('serviceStop')
-                              : capitalizeFirstLetter(
-                                  t(
-                                    `stdcm:trainPath.stopType.${stdcmData.simulationPathSteps[renderedIndex - 1].stopType}`
-                                  )
-                                )}
-                          </TD>
-                        </View>
-                      </TR>
-                    );
-                  }
-                  return null;
+                  const isLastStep = index === stdcmData.simulationPathSteps.length - 1;
+                  renderedIndex += 1;
+                  return (
+                    <TR key={index} style={styles.convoyAndRoute.stopTableTbody}>
+                      <View style={styles.convoyAndRoute.stopTableIndexWidth}>
+                        <TD style={styles.convoyAndRoute.stopTableIndexColumn}>{renderedIndex}</TD>
+                      </View>
+                      <View style={styles.convoyAndRoute.stopTableOpWidth}>
+                        <TD style={styles.convoyAndRoute.stopTableOpColumn}>
+                          {step.name || 'Unknown'}
+                        </TD>
+                      </View>
+                      <View style={styles.convoyAndRoute.stopTableChWidth}>
+                        <TD style={styles.convoyAndRoute.stopTableChColumn}>{step.ch}</TD>
+                      </View>
+                      <View style={styles.convoyAndRoute.stopTableEndWidth}>
+                        <TD style={styles.convoyAndRoute.stopTableItalicColumn}>
+                          {getArrivalTime(step, { isFirstStep: false, isLastStep })}
```

Comment:
Once isFirstStep and isLastStep are nullable, you can simply remove the false:
```suggestion
                          {getArrivalTime(step, { isLastStep })}
```

Reply:
fixed